ui/backend/server/server/settings.py

```diff
@@ -223,7 +223,7 @@ def get_from_env(
         }
     }
     LOGGING["handlers"]["console"]["filters"] = ["truncate_sql"]
-    SILKY_PYTHON_PROFILER = True
-    MIDDLEWARE.insert(0, "silk.middleware.SilkyMiddleware")
+    SILKY_PYTHON_PROFILER = False
+    # MIDDLEWARE.insert(0, "silk.middleware.SilkyMiddleware")
     MIDDLEWARE.append("trackingserver_base.middleware.timing_middleware.TimingMiddleware")
-    INSTALLED_APPS.append("silk")
+    # INSTALLED_APPS.append("silk")
```

Comment:
why comments though?

Reply:
It'll break but I want to keep it around to show